packages/amplify-cli-core/API.md

```diff
@@ -2131,6 +2139,12 @@ export interface Template {
 // @public (undocumented)
 export type TimedCodePath = ManuallyTimedCodePath | UntilExitTimedCodePath | FromStartupTimedCodePaths;
 
+// @public (undocumented)
+export const toPackageManagerInstallArgs: (packageManager: PackageManager, buildType?: BuildType) => Promise<string[]>;
+
+// @public (undocumented)
+export const toPackageManagerRunScriptArgs: (packageManager: PackageManager, scriptName: string) => Promise<string[]>;
```

Comment:
should these be methods on a PackageManager class?

Reply:
converted to class methods